modules/openthread/platform/platform-zephyr.h

```diff
@@ -77,6 +77,12 @@ uint16_t platformRadioChannelGet(otInstance *aInstance);
 otError platformRadioTransmitCarrier(otInstance *aInstance, bool aEnable);
 #endif /* CONFIG_IEEE802154_CARRIER_FUNCTIONS */
 
+/**
+ * Start/stop modulated carrier wave transmission.
+ */
+otError platformRadioTransmitModulatedCarrier(otInstance *aInstance, bool aEnable,
+					      const uint8_t *aData);
+
```

Comment:
bit. This should be wrapped with
#if defined(CONFIG_IEEE802154_CARRIER_FUNCTIONS)
#endif
as the implementation does.

Reply:
fixed.